biomage/stage/stage.py

```diff
@@ -622,11 +652,21 @@ def stage(token, org, deployments):
     click.echo(
         click.style(
             "✔️ Deployment submitted. You can check your progress at "
-            f"https://github.com/{org}/iac/actions",
+            f"https://github.com/{org}/iac/actions. When the deployment is done"
+            " run the following command to trigger flux synchronization and "
+            " speed up the process:",
             fg="green",
             bold=True,
         )
     )
+
+    click.echo()
+    click.echo(
+        "fluxctl sync --k8s-fwd-ns flux --context arn:aws:eks:eu-west-1:"
+        "242905224710:cluster/biomage-staging",
+    )
```

Comment:
imo it would be neat to indent this, feel free to ignore if you don't see the point

Reply:
ok